modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/freelist/FreeListImpl.java

```diff
@@ -886,9 +614,28 @@ public long recycledPagesCount() throws IgniteInternalCheckedException {
         }
     }
 
-    /** {@inheritDoc} */
+    @Override
+    public void saveMetadata() throws IgniteInternalCheckedException {
+        saveMetadata(statHolder);
+    }
+
+    /** Returns page eviction tracker. */
+    public PageEvictionTracker evictionTracker() {
+        return evictionTracker;
+    }
+
+    /** Returns page memory. */
+    public PageMemory pageMemory() {
+        return pageMem;
+    }
+
+    /** Returns write row handler. */
+    public WriteRowHandler writeRowHandler() {
```

Comment:
This is weird, why did you make it a method? Artifact of the refactoring?
I would recommend you to rollback some of the changes.